mysql-test/suite/rocksdb/include/type_varchar_endspace.inc

```diff
@@ -42,18 +42,24 @@ insert into t1 values (0, 'ab', 'a-b');
 insert into t1 values (1, 'a ', 'a-space');
 insert into t1 values (2, 'a',  'a');
 insert into t1 values (3, 'a  \t', 'a-tab');
+# force cardinality updates otherwise explains below are non-deterministic
```

Comment:
No need for such comment (when a testcase does CREATE TABLE ... INSERT ... ANALYZE ... [EXPLAIN ]SELECT,  the role of ANALYZE is always clear).

If this were the only review comment then it could have stayed